_includes/footer.html

```diff
@@ -0,0 +1,20 @@
+<!-- footer -->
+<footer class="bg-dark footer-section">
+  <div class="section">
+    <div class="container">
+      <div class="row">
+        <div class="col-md-12 text-center">
+          <h5 class="text-light">Email</h5>
+          <p class="text-white paragraph-lg font-secondary">{{ site.data.settings.email }}</p>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="border-top text-center border-dark py-5">
+    <p class="mb-0 text-light">Copyright ©<script>
```

Comment:
This may be a nitpick, but technically our project is MIT. :3 So claiming all rights may not be what we want. ^w^ An option would be to declare it as such: `Licensed under MIT.` with a [link to the MIT license](https://choosealicense.com/licenses/mit/). An option may also be to add a copy heart instead of the copyright symbol, [as demonstrated here](http://copyheart.org/).

Reply:
Fixed